File: app/src/main/java/com/example/learnit/MainActivity.java
package com.example.learnit;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.TextView;


public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        TextView colors= findViewById(R.id.colors);
        colors.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent colorsIntent= new Intent(MainActivity.this, colors.class);
                startActivity(colorsIntent);
            }
        });
        TextView family=findViewById(R.id.family);
        family.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent familyIntent=new Intent(MainActivity.this,family.class);
                startActivity(familyIntent);
            }
        });
        TextView numbers= findViewById(R.id.numbers);
        numbers.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent numbersIntent = new Intent(MainActivity.this,numbers.class);
                startActivity(numbersIntent);
            }
        });
        final TextView phrases=findViewById(R.id.phrases);
        phrases.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent phrasesIntent=new Intent(MainActivity.this , phrases.class);
                startActivity(phrasesIntent);
            }
        });

    }
}

File: app/src/main/java/com/example/learnit/MainActivity.kt
package com.example.learnit

import android.content.Intent
import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val colors = findViewById<TextView>(R.id.colors)
        colors.setOnClickListener {
            val colorsIntent = Intent(this@MainActivity, com.example.learnit.colors::class.java)
            startActivity(colorsIntent)
        }
        val family = findViewById<TextView>(R.id.family)
        family.setOnClickListener {
            val familyIntent = Intent(this@MainActivity, com.example.learnit.family::class.java)
            startActivity(familyIntent)
        }
        val numbers = findViewById<TextView>(R.id.numbers)
        numbers.setOnClickListener {
            val numbersIntent = Intent(this@MainActivity, com.example.learnit.numbers::class.java)
            startActivity(numbersIntent)
        }
        val phrases = findViewById<TextView>(R.id.phrases)
        phrases.setOnClickListener {
            val phrasesIntent = Intent(this@MainActivity, com.example.learnit.phrases::class.java)
            startActivity(phrasesIntent)
        }
    }
}